.gitattributes

```diff
@@ -2,3 +2,8 @@
 # and treat them like normal text.
 *.min.js binary
 *.min.css binary
+
+# The profile conversion test includes very verbose snapshots that make code diff
```

Comment:
This is working locally but not in github. Perhaps it needs to be merged into `main`? Let me know if you want me to remove it.

Reply:
-diff seems to work locally.